common/remote_object.go

```diff
@@ -40,17 +39,62 @@ func (pe *objectPropertyParseError) Unwrap() error {
 	return pe.error
 }
 
+type multiError struct {
+	Errors []error
+}
+
+func (me *multiError) append(err error) {
+	me.Errors = append(me.Errors, err)
+}
+
+func (me multiError) Error() string {
```

Comment:
Can you also add a test for this `Error` function?

Reply:
Sure, done in [c6e7e9a](https://github.com/grafana/xk6-browser/pull/734/commits/c6e7e9ac8cb3e3b610ffbb2905d8b88c4c444494).